Specs/Renderer/CubeMapSpec.js

```diff
@@ -763,155 +453,98 @@ defineSuite([
-        expect(context.readPixels()).toEqual([255, 0, 0, 0]);
-
-        sp.allUniforms.u_direction.value = new Cartesian3(0, 0, 1);
-        command.execute(context);
-        expect(context.readPixels()).toEqual([0, 0, 255, 255]);
-
-        sp.allUniforms.u_direction.value = new Cartesian3(0, 0, -1);
-        command.execute(context);
-        expect(context.readPixels()).toEqual([255, 255, 0, 0]);
     });
 
     it('copies from the framebuffer', function() {
+        var cxt = createContext({
+            webgl : {
+                alpha : true // Seems to be required for copyFromFramebuffer()
+            }
+        });
+
         cubeMap = new CubeMap({
-            context : context,
+            context : cxt,
             width : 1,
             height : 1
         });
         cubeMap.positiveX.copyFrom(blueImage);
 
-        var vs = 'attribute vec4 position; void main() { gl_PointSize = 1.0; gl_Position = position; }';
         var fs =
             'uniform samplerCube u_cubeMap;' +
             'void main() { gl_FragColor = textureCube(u_cubeMap, vec3(1.0, 0.0, 0.0)); }';
 
-        sp = ShaderProgram.fromCache({
-            context : context,
-            vertexShaderSource : vs,
-            fragmentShaderSource : fs,
-            attributeLocations : {
-                position : 0
+        var uniformMap = {
+            u_cubeMap : function() {
+                return cubeMap;
             }
-        });
-
-        sp.allUniforms.u_cubeMap.value = cubeMap;
-
-        va = new VertexArray({
-            context : context,
-            attributes : [{
-                vertexBuffer : Buffer.createVertexBuffer({
-                    context : context,
-                    typedArray : new Float32Array([0, 0, 0, 1]),
-                    usage : BufferUsage.STATIC_DRAW
-                }),
-                componentsPerAttribute : 4
-            }]
-        });
-
-        var command = new DrawCommand({
-            primitiveType : PrimitiveType.POINTS,
-            shaderProgram : sp,
-            vertexArray : va
-        });
+        };
 
         // +X is blue
-        command.execute(context);
-        expect(context.readPixels()).toEqual([0, 0, 255, 255]);
+        expect({
+            context : cxt,
+            fragmentShader : fs,
+            uniformMap : uniformMap
+        }).contextToRender([0, 0, 255, 255]);
 
         // Clear framebuffer to red and copy to +X face
         var clearCommand = new ClearCommand({
             color : new Color (1.0, 0.0, 0.0, 1.0)
         });
 
-        clearCommand.execute(context);
-        expect(context.readPixels()).toEqual([255, 0, 0, 255]);
+        clearCommand.execute(cxt);
+        expect(cxt).toReadPixels([255, 0, 0, 255]);
         cubeMap.positiveX.copyFromFramebuffer();
 
-        ClearCommand.ALL.execute(context);
-        expect(context.readPixels()).toEqual([0, 0, 0, 0]);
+        ClearCommand.ALL.execute(cxt);
+        expect(cxt).toReadPixels([0, 0, 0, 0]);
```

Comment:
It is because `alpha : true` is used when creating the context.

I cleaned this up in this PR; when I first wrote the tests....6 years ago...for some reason I had the tests default to `alpha : true` even though Cesium did not (or maybe it did, then I set it to `false` because I heard it made browser compositing faster).  Anyway, I think these are all correct.